packages/leancode_hooks/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: leancode_hooks
-version: 0.0.6
+version: 0.1.0
 homepage: https://github.com/leancodepl/flutter_corelibrary/tree/master/packages/leancode_hooks
 repository: https://github.com/leancodepl/flutter_corelibrary
 description: >-
   Hooks we often use, all gathered in one place.
 
 environment:
-  sdk: '>=3.0.0 <4.0.0'
-  flutter: '>=3.10.0 <4.0.0'
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.10.0 <4.0.0"
```

Comment:
I think the default style would be single quotes

Reply:
Currently, both styles are used in this file, the autoformatter just normalized it.

Reply:
If that's _the default_, is there any reference for what is considered a default? So, I can at least update my autoformatter

Reply:
I think Dart uses single quotes by default, if we were to normalize it it should be into single quotes

Reply:
As for the reference, I just looked at yamls in the repos of Dart and Flutter

Reply:
I think we may be looking at different repos then? [go_router](https://github.com/flutter/packages/blob/16616ad588d7ad9c00958ec61022e5b5dde5f5dd/packages/go_router/pubspec.yaml#L10), [google_sign_in](https://github.com/flutter/packages/blob/16616ad588d7ad9c00958ec61022e5b5dde5f5dd/packages/google_sign_in/google_sign_in/pubspec.yaml#L10), [image_picker](https://github.com/flutter/packages/blob/16616ad588d7ad9c00958ec61022e5b5dde5f5dd/packages/image_picker/image_picker/pubspec.yaml#L10), [shared_preferences](https://github.com/flutter/packages/blob/16616ad588d7ad9c00958ec61022e5b5dde5f5dd/packages/shared_preferences/shared_preferences/pubspec.yaml#L10), [etc.](https://github.com/search?q=repo%3Aflutter%2Fpackages+%22flutter%3A%22+language%3AYAML&type=code&l=YAML)

Reply:
Fwiw, if you generate a fresh Flutter project, the pubspec will use single quotes